### PR TITLE
Next gen

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,83 @@
...
 
-Warning: If you previously ran  ```npm install``` locally, the node_modules folder will conflict with the file structure of the container when you run the above command.
-We recommended installing your node dependencies one folder up from the rest of your source code.
+1. To stop your containers, run `docker-compose stop`.
 
-You can then visit the application at [http://localhost:3000/](http://localhost:3000/). If you're using [boot2docker](https://docs.docker.com/installation/mac/) to run Docker on OS X, you'll need to use the value of `boot2docker ip` instead of `localhost`.
+## Deploy On Heroku
 
-To stop your containers, run `docker-compose stop`.
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/TwilioDevEd/lead-alerts-node)
```
Comment (acamino) [inline README.md]: Probably this is the right time to use the **.svg** version of the Heroku button.

File: README.md [hunk 2]
```diff
@@ -1,66 +1,83 @@
...
-- `TWILIO_NUMBER`
-- `TWILIO_ACCOUNT_SID`
-- `TWILIO_AUTH_TOKEN`
-- `AGENT_NUMBER`
+  ```
+  $ docker-compose up -d
+  ```
 
-Finally, run the following commands to start your Docker containers:
+  Warning: If you previously ran  ```npm install``` locally, the node_modules folder will conflict with the file structure of the container when you run the above command.
```
Comment (acamino) [inline README.md]: Extra space at the beginning of this line. Would you mind removing it please?

File: README.md [hunk 3]
```diff
@@ -1,66 +1,83 @@
...
+  We recommended installing your node dependencies one folder up from the rest of your source code.
```
Comment (acamino) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -1,66 +1,83 @@
...
+  $ npm install -g nodemon
+  $ nodemon .
+  ```
 
 ### Docker local development
 
-First, install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+1. Install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+1. Make sure you performed step `2` on the previous section.
```
Comment (acamino) [inline README.md]: What do you think about?

.. step **2** shown in the previous section.

I do prefer to use back ticks for highlighting code or something related.

**Review by acamino — CHANGES_REQUESTED** [review]
Thanks @hortega! This one is almost ready. Keep up with the good work.

File: README.md (hunk 4 above)
Comment (hortega) [reply README.md]: I agree, it's more clear in bold

File: README.md (hunk 2 above)
Comment (hortega) [reply README.md]: Those spaces are indenting the paragraph as it's in an ordered list, it would look awkward without don't you think?

**Review by acamino — APPROVED** [review]

**Comment by acamino** [comment]
Thanks @hortega!

**Pull request merged**